web/app/fr/gospeak/web/pages/orga/contacts/detail.scala.html

```diff
@@ -0,0 +1,30 @@
+@import com.mohiva.play.silhouette.api.actions.SecuredRequest
+@import fr.gospeak.core.domain.{Group, Partner, User}
+@import fr.gospeak.web.auth.domain.CookieEnv
+@import fr.gospeak.web.domain.Breadcrumb
+@import fr.gospeak.web.pages.orga.partials.html.container
+@import fr.gospeak.web.pages.orga.partners.routes.PartnerCtrl
+@import fr.gospeak.web.pages.partials.display.html._
+
+@import fr.gospeak.core.domain.Contact
+@(group: Group, partner: Partner, contact: Contact, speakers: Seq[User], edit: Call)(b: Breadcrumb)(implicit req: SecuredRequest[CookieEnv, AnyContent], messages: Messages)
+@container(partner.name.value + " - Gospeak", group, b)() {
+    <div class="d-flex justify-content-between align-items-start">
+        <div class="media">
+            <img src="@partner.logo.value" class="avatar align-self-start mr-3" alt="avatar">
+            <div class="media-body">
+                <h1 class="mt-0"><a href="@PartnerCtrl.detail(group.slug, partner.slug)">@partner.name.value</a></h1>
+                <p>@contact.email.value</p>
+
+                <p>@(contact.firstName.value)</p>
+
+                <p>@(contact.lastName.value)</p>
+
+                @if(contact.description.isDefined) {
```

Comment:
You can do: `@contact.description.map { d => <p>@d.value</p> }`, Play manage well collections and options in templates